src/core/config/OperationConfig.js

```diff
@@ -2261,6 +2261,20 @@ const OperationConfig = {
             }
         ]
     },
+    "From Windows Filetime To UNIX":{
+        description: "Converts a Windows Filetime timestamp to a datetime format",
+        run: DateTime.runFromFiletimeToUnix,
+        inputType: "string",
+        outputType: "string",
+        args: []
```

Comment:
UNIX timestamps are sometimes expressed in seconds and sometimes in milliseconds. It would be nice to have an argument for this operation that allows the user to choose the output units (like in [the other UNIX timestamp operations](https://github.com/gchq/CyberChef/blob/master/src/core/operations/DateTime.js#L16)).